.github/workflows/pull-request-review.yml

```
@@ -0,0 +1,46 @@
+name: Pull Request Review
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)